### PR TITLE
[ja] Translate tasks/debug-application-cluster/crictl into Japanese

### PR DESCRIPTION
**Pull request by ptux**
Address #30869

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @nasa9084

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
+---
+reviewers:
+- ptux
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:3]: please remove reviewers block

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.11" state="stable" >}}
+
+`crictl`はCRI互換のコンテナランタイム用のコマンドラインインターフェイスです。
+
+これを使って、Kubernetes ノード上のコンテナランタイムやアプリケーションの検査やデバッグを行うことができます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:15]:
```suggestion
これを使って、Kubernetesノード上のコンテナランタイムやアプリケーションの検査やデバッグを行うことができます。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+`crictl`とそのソースは[cri-tools](https://github.com/kubernetes-sigs/cri-tools)リポジトリにホストされています。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:16]:
```suggestion
`crictl`とそのソースコードは[cri-tools](https://github.com/kubernetes-sigs/cri-tools)リポジトリにホストされています。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+
+## {{% heading "prerequisites" %}}
+
+
+`crictl`にはCRIランタイムを搭載した Linux が必要です。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:21]:
```suggestion
`crictl`にはCRIランタイムを搭載したLinuxが必要です。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 5]
```diff
@@ -0,0 +1,409 @@
...
+
+
+
+<!-- steps -->
+
+## crictlのインストール
+
+cri-tools [release page](https://github.com/kubernetes-sigs/cri-tools/releases)から、いくつかの異なるアーキテクチャ用の圧縮アーカイブ`crictl`をダウンロードできます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:29]:
```suggestion
cri-toolsの[リリースページ](https://github.com/kubernetes-sigs/cri-tools/releases)から、いくつかの異なるアーキテクチャ用の圧縮アーカイブ`crictl`をダウンロードできます。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 6]
```diff
@@ -0,0 +1,409 @@
...
+
+お使いのKubernetesのバージョンに対応するバージョンをダウンロードしてください。
+それを解凍してシステムパス上の`/usr/local/bin/`などの場所に移動します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:32]:
```suggestion
それを解凍して`/usr/local/bin/`などのシステムパス上の場所に移動します。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 7]
```diff
@@ -0,0 +1,409 @@
...
+timeout: 10
+debug: true
+```
+
+## crictlコマンドの例
+
+以下の例では、いくつかの`crictl`コマンドとその出力例を示しています。
+
+{{< warning >}}
+実行中のKubernetesクラスターに`crictl`を使ってポッドのサンドボックスやコンテナを作成しても、Kubeletは最終的にそれらを削除します。`crictl` は汎用のワークフローツールではなく、デバッグに便利なツールです。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:65]:
```suggestion
実行中のKubernetesクラスターに`crictl`を使ってPodのサンドボックスやコンテナを作成しても、Kubeletは最終的にそれらを削除します。`crictl` は汎用のワークフローツールではなく、デバッグに便利なツールです。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 8]
```diff
@@ -0,0 +1,409 @@
...
+{{< /warning >}}
+
+### podsの一覧
+
+すべてのポッドをリストアップ:
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:70]:
```suggestion
すべてのPodをリストアップ:
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 9]
```diff
@@ -0,0 +1,409 @@
...
+
+```
+CONTAINER ID        IMAGE                                                                                                             CREATED             STATE               NAME                       ATTEMPT
+1f73f2d81bf98       busybox@sha256:141c253bc4c3fd0a201d32dc1f493bcf3fff003b6df416dea4f41046e0f37d47                                   7 minutes ago       Running             sh                         1
+9c5951df22c78       busybox@sha256:141c253bc4c3fd0a201d32dc1f493bcf3fff003b6df416dea4f41046e0f37d47                                   8 minutes ago       Exited              sh                         0
+87d3992f84f74       nginx@sha256:d0a8828cccb73397acb0073bf34f4d7d8aa315263f1e7806bf8c55d8ac139d5f                                     8 minutes ago       Running             nginx                      0
+1941fb4da154f       k8s-gcrio.azureedge.net/hyperkube-amd64@sha256:00d814b1f7763f4ab5be80c58e98140dfc69df107f253d7fdd714b30a714260a   18 hours ago        Running             kube-proxy                 0
+```
+
+ランニングコンテナをリストアップします:
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:176]:
```suggestion
実行中のコンテナをリストアップします:
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 10]
```diff
@@ -0,0 +1,409 @@
...
+出力はこのようになります:
+
+```
+10.240.0.96 - - [06/Jun/2018:02:45:51 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.47.0" "-"
+```
+
+### Podサンドボックスの実行
+
+`crictl`を使ってPodサンドボックスを実行することは、コンテナのランタイムをデバッグするのに便利です。
+稼働中のKubernetesクラスタでは、サンドボックスは最終的にKubeletによって停止され、削除されます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:234]:
```suggestion
稼働中のKubernetesクラスターでは、サンドボックスは最終的にKubeletによって停止され、削除されます。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 11]
```diff
@@ -0,0 +1,409 @@
...
+
+1. busyboxイメージをプルします:
+
+      ```shell
+      crictl pull busybox
+      Image is up to date for busybox@sha256:141c253bc4c3fd0a201d32dc1f493bcf3fff003b6df416dea4f41046e0f37d47
+      ```
+
+2. Podとコンテナのコンフィグを作成します:
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:272]:
```suggestion
2. Podとコンテナの設定を作成します:
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 12]
```diff
@@ -0,0 +1,409 @@
...
+      }
+      ```
+
+3. 先に作成されたPodのID、コンテナの設定ファイル、Podの設定ファイルを渡して、コンテナを作成します。コンテナのIDが返されます。
+
+      ```shell
+      crictl create f84dd361f8dc51518ed291fbadd6db537b0496536c1d2d6c05ff943ce8c9a54f container-config.json pod-config.json
+      ```
+
+4.  すべてのコンテナをリストアップし、新しく作成されたコンテナのステートが`Created`に設定されていることを確認します:
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:313]:
```suggestion
4.  すべてのコンテナをリストアップし、新しく作成されたコンテナの状態が`Created`に設定されていることを確認します:
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 13]
```diff
@@ -0,0 +1,409 @@
...
+{{< note >}}
+CRICTLの出力形式はDocker CLIと似ていますが、いくつかのCLIでは列が欠けています。
+{{< /note >}}
+
+### デバッグ情報の取得
+
+{{< table caption="mapping from docker cli to crictl - retrieve debugging information" >}}
+docker cli | crictl | 説明 | サポートされていない機能
+-- | -- | -- | --
+`attach` | `attach` | ランニングコンテナにアタッチ | `--detach-keys`, `--sig-proxy`
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:371]:
```suggestion
`attach` | `attach` | 実行中のコンテナにアタッチ | `--detach-keys`, `--sig-proxy`
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 14]
```diff
@@ -0,0 +1,409 @@
...
+`exec` | `exec` | ランニングコンテナにコマンドの実行 | `--privileged`, `--user`, `--detach-keys`
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:372]:
```suggestion
`exec` | `exec` | 実行中のコンテナでコマンドの実行 | `--privileged`, `--user`, `--detach-keys`
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 15]
```diff
@@ -0,0 +1,409 @@
...
+`images` | `images` | イメージのリストアップ |  
+`info` | `info` | システム全体の情報の表示 |  
+`inspect` | `inspect`, `inspecti` | コンテナ、イメージ、タスクの低レベルの情報を返します |  
+`logs` | `logs` | コンテナのログを取得します | `--details`
+`ps` | `ps` | コンテナのリストアップ |  
+`stats` | `stats` | コンテナのリソース使用状況をライブで表示 | Column: NET/BLOCK I/O, PIDs
+`version` | `version` | ランタイム（Docker、ContainerD、その他）のバージョン情報を表示します |
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:379]:
```suggestion
`version` | `version` | ランタイム(Docker、ContainerD、その他)のバージョン情報を表示します |  
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md [hunk 16]
```diff
@@ -0,0 +1,409 @@
...
+{{< /table >}}
+
+### 変更を行います
+
+{{< table caption="mapping from docker cli to crictl - perform changes" >}}
+docker cli | crictl | 説明 | サポートされていない機能
+-- | -- | -- | --
+`create` | `create` | 新しいコンテナを作成します |  
+`kill` | `stop` (timeout = 0) | 1つ以上の実行中のコンテナを停止します | `--signal`
+`pull` | `pull` | レジストリからイメージやリポジトリをプルします | `--all-tags`, `--disable-content-trust`
```
Comment (nasa9084) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:389]:
```suggestion
`pull` | `pull` | レジストリーからイメージやリポジトリをプルします | `--all-tags`, `--disable-content-trust`
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

File: content/ja/docs/tasks/debug-application-cluster/crictl.md (hunk 7 above)
Comment (riita10069) [reply content/ja/docs/tasks/debug-application-cluster/crictl.md:65]:
```suggestion
実行中のKubernetesクラスターに`crictl`を使ってPodのサンドボックスやコンテナを作成しても、Kubeletは最終的にそれらを削除します。`crictl`は汎用のワークフローツールではなく、デバッグに便利なツールです。
```

File: content/ja/docs/tasks/debug-application-cluster/crictl.md (hunk 12 above)
Comment (riita10069) [inline content/ja/docs/tasks/debug-application-cluster/crictl.md:313]:
```suggestion
4. すべてのコンテナをリストアップし、新しく作成されたコンテナの状態が`Created`に設定されていることを確認します:
```

**Pull request merged**